### PR TITLE
Features/new help

### PR DESCRIPTION
**Pull request by Mikanebu**
* removed `hubot-help` from `package.json`
* created `docs/help.md` - markdown with help messages
  * added section `tags`
  * added section `commands`
* refactored main.js to use `docs/help.md`
* removed unused packages from `external-scripts.json` and from `package.json`

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -12,24 +12,24 @@
     "babel-register": "^6.24.0",
     "coffee-script": "1.8.0",
     "dotenv": "^4.0.0",
+    "fs": "0.0.1-security",
```
Comment (zelima) [inline package.json]: fs is built in library, no?

File: docs/help.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+  - *title* - can accept the following values:
+    - any date in the format of `DD MMM YYYY`
+    - any string
+    For example: "13 Jan 2017" or "This is Backlog"
+    
+`bot issue "title" about "body" in "myorg/myrepo"` - Create an issue on github
+  - *title* - the title of the issue
+  - *body* - the contents of the issue
+
+`bot links` - Return gdocs url for tag `+link`
```
Comment (zelima) [inline docs/help.md]: Gdocs or Google Docs

Comment (zelima) [reply docs/help.md]: Actually it's Google Spreadsheets as much as I remember

File: docs/help.md [hunk 3]
```diff
@@ -0,0 +1,27 @@
+### Commands
+`bot adapter` - Reply with the adapter
```
Comment (zelima) [inline docs/help.md:2]: What is the adapter?

File: docs/help.md [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+
+`bot create|close milestone all "title"` - Create or close all milestones listed in `config.json`
```
Comment (zelima) [inline docs/help.md]: think it not creates all milestones listed in `config.json`, but creates milestone for **all repos** listed in `config.json`

File: docs/help.md [hunk 5]
```diff
@@ -0,0 +1,27 @@
...
+    For example: "13 Jan 2017" or "This is Backlog"
+    
+`bot issue "title" about "body" in "myorg/myrepo"` - Create an issue on github
+  - *title* - the title of the issue
+  - *body* - the contents of the issue
+
+`bot links` - Return gdocs url for tag `+link`
+`bot map me <query>` - Returns a map view of the area returned by query.
+`bot the rules` - Make sure hubot still knows the rules.
+`bot todos` - Return gdocs url for tag `+todo`
```
Comment (zelima) [inline docs/help.md]: same here - it's google spreadsheet not google doc

File: docs/help.md [hunk 6]
```diff
@@ -0,0 +1,27 @@
...
+`bot time` - Reply with current time
+`ship it` - Display a motivation squirrel
+`!thank user` - Motivate user 
+
+### Tags
+In your chat channel tag at start, middle or end of a message: `+todo`,`+link`
+
+
+
```
Comment (zelima) [inline docs/help.md]: Why 3 whitespaces?

**Review by zelima — CHANGES_REQUESTED** [review]
Looks goog, small comments

File: package.json (hunk 1 above)
Comment (Mikanebu) [reply package.json]: I do know why, but when I run `hubot` locally, it asked me for `fs` library.

File: docs/help.md (hunk 3 above)
Comment (Mikanebu) [reply docs/help.md:2]: It is just small `hubot` feature that returns adapter. In our case it is gitter adapter)

File: docs/help.md (hunk 4 above)
Comment (Mikanebu) [reply docs/help.md]: will be fixed

File: docs/help.md (hunk 2 above)
Comment (Mikanebu) [reply docs/help.md]: Google Docs is a free Web-based application in which documents and spreadsheets can be created. We can use google spreadsheet to clarify

File: package.json (hunk 1 above)
Comment (zelima) [reply package.json]: I would try and debug this on your local machine. Don't think this is general bug and others need to install it as well

File: docs/help.md (hunk 2 above)
Comment (zelima) [reply docs/help.md]: Yes please, let's use spreadsheet

File: package.json (hunk 1 above)
Comment (Mikanebu) [reply package.json]: @zelima COOOL! works fine, removed `fs`

File: docs/help.md [hunk 7]
```diff
@@ -1,7 +1,7 @@
 ### Commands
 `bot adapter` - Reply with the adapter
 
-`bot create|close milestone all "title"` - Create or close all milestones listed in `config.json`
+`bot create|close milestone all "title"` - Creates milestone for **all repos** listed in `config.json`
```
Comment (zelima) [inline docs/help.md]: Why making it bold?

File: docs/help.md [hunk 8]
```diff
@@ -12,16 +12,13 @@
   - *title* - the title of the issue
   - *body* - the contents of the issue
 
-`bot links` - Return gdocs url for tag `+link`
+`bot links` - Return google spreadsheet url for tag `+link`
```
Comment (zelima) [inline docs/help.md]: think Google SpreadSheet (Camelcased) looks better

File: docs/help.md (hunk 7 above)
Comment (Mikanebu) [reply docs/help.md]: @zelima in your comment, it was bold, or you just showed me?

File: docs/help.md (hunk 8 above)
Comment (Mikanebu) [reply docs/help.md]: will be fixed

**Review by zelima — APPROVED** [review]

**Pull request merged**